photon-core/src/main/java/org/photonvision/vision/frame/provider/USBFrameProvider.java

```diff
@@ -43,9 +46,9 @@ public CapturedFrame getInputMat() {
                 cvSink.grabFrame(mat.getMat())
                         * 1000; // Units are microseconds, epoch is the same as the Unix epoch
 
-        // Sometimes CSCore gives us a zero frametime.
-        if (time <= 1e-6) {
-            time = MathUtils.wpiNanoTime();
+        if (time == 0) {
```

Comment:
guaranteed to be 0 on error, might want to comment that here: https://first.wpi.edu/wpilib/allwpilib/docs/release/java/edu/wpi/first/cscore/CvSink.html#grabFrame(org.opencv.core.Mat)